packages/core/src/cache/cache.ts

```diff
@@ -168,19 +167,11 @@ class Cache implements ICache {
       volume.imageData.delete();
     }
 
-    // if we had views for the images of the volume, we need to restore them
-    // to avoid memory leaks
-    restoreImagesFromBuffer(volume);
```

Comment:
It feels a lot better not to be restoring the images - that always felt kind of dangerous.